Core/NoOpRunner.Core/GameSettings.cs

```diff
@@ -13,7 +13,7 @@ public  static class GameSettings
         
         public const int TimeBetweenFramesMs = 70;
 
-        public static int TimeBetweenMapStep => 1000 / 70;
+        public static int TimeBetweenMapStep => 500 / 70;
```

Comment:
Should be even lower imo

Reply:
I agree, but a bit later. At least let's wait until we fix the sync issues. This should be enough for demoing.